test/lambda_toolshed/papillon_test.cljc

```diff
@@ -126,9 +126,9 @@
         ixs [capture-ix
              {:name :thrower :enter (fn [_] (throw the-exception))}]
         expected-log [[:capture :enter]
-	              [:thrower :enter]
-	              [:thrower :error]
-	              [:capture :error]]]
+                      [:thrower :enter]
```

Comment:
I cannot fathom how I introduced tabs here.  Emacs is not configured to use tabs *and* I have editorconfig globally set to not use them in clj.